### PR TITLE
[tech] Add offers for NeTEx France

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/netex_france/offer.rs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+use transit_model_collection::Idx;
+
+#[derive(Debug)]
+enum ObjectType {
+    _Route,
+    _ServiceJourneyPattern,
+    _ScheduledStopPoint,
+    _PassengerStopAssignment,
+    _ServiceJourney,
+}
```
Comment (woshilapin) [inline src/netex_france/offer.rs]: :warning: This part might need to be refactor if #546 is merged first.

**Comment by woshilapin** [comment]
I updated the PR with all the functionality for creating offers' files.

File: src/netex_france/offer.rs [hunk 2]
```diff
@@ -119,6 +122,46 @@ impl<'a> OfferExporter<'a> {
...
+        vehicle_journey_id: &'a str,
+        stop_time: &'a StopTime,
+    ) -> Element {
+        Element::builder(ObjectType::StopPointInJourneyPattern.to_string())
+            .attr(
+                "id",
+                Self::generate_stop_sequence_id(
+                    &vehicle_journey_id,
+                    stop_time.sequence,
+                    ObjectType::ScheduledStopPoint,
```
Comment (ArnaudOggy) [inline src/netex_france/offer.rs]:
```suggestion
                    ObjectType::StopPointInJourneyPattern,
```

Comment (woshilapin) [reply src/netex_france/offer.rs]: Done and I updated the fixtures with it.

File: src/netex_france/offer.rs [hunk 3]
```diff
@@ -314,6 +382,36 @@ impl<'a> OfferExporter<'a> {
             .build()
     }
 
+    fn generate_day_type_ref(vehicle_journey_id: &'a str) -> Element {
```
Comment (ArnaudOggy) [inline src/netex_france/offer.rs]:
```suggestion
    fn generate_day_type_ref(service_id: &'a str) -> Element {
```

Comment (woshilapin) [reply src/netex_france/offer.rs]: Thanks, fix in the last commit.

**Pull request merged**